internal/cmd/extensions/client.go

```diff
@@ -65,7 +67,7 @@ func (e *E11yClient) Delete(ctx context.Context, companyID string, extensionID s
 		return fmt.Errorf("error executing request: %w", err)
 	}
 
-	if resp.StatusCode() > 399 {
+	if resp.StatusCode() > minimalSuccessStatusCode {
```

Comment:
we can use http consts
```suggestion
	if resp.StatusCode() >= http.StatusBadRequest {
```

Reply:
makes sense lol